src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
 
+        List<String> lines = new ArrayList<>();
```

Comment:
![image](https://github.com/mate-academy/jv-work-with-file/assets/20326858/a2889627-f819-468f-9f3d-5fc43d42b815)

```
String dataFromFile = readFile(fromFileName);
String report = createReport(dataFromFile);
writeToFile(report, toFileName);
```

Reply:
Can You please explan this alittle bit ? @Rommelua

Reply:
Write to java chat anytime when you need additional explanation please. PR is not for long discussion)